common/concurrency/limits/src/main/java/io/helidon/common/concurrency/limits/ThroughputLimitConfigBlueprint.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.common.concurrency.limits;
+
+import java.time.Duration;
+import java.util.Optional;
+import java.util.concurrent.Semaphore;
+import java.util.function.Supplier;
+
+import io.helidon.builder.api.Option;
+import io.helidon.builder.api.Prototype;
+import io.helidon.common.concurrency.limits.spi.LimitProvider;
+
+/**
+ * Configuration of {@link ThroughputLimit}.
+ *
+ * @see #amount()
+ * @see #duration()
+ * @see #queueLength()
+ * @see #queueTimeout()
+ */
+@Prototype.Blueprint
+@Prototype.Configured(value = ThroughputLimit.TYPE, root = false)
+@Prototype.Provides(LimitProvider.class)
+interface ThroughputLimitConfigBlueprint extends Prototype.Factory<ThroughputLimit> {
+    String TOKEN_BUCKET = "token-bucket";
```

Comment:
These constants are visible through implemented interfaces. Please move them to a package local class.
If they should be public on the generated prototype, you can add them from custom methods type using `@Prototype.Constant`